### PR TITLE
Feature: Video Producer

### PR DESCRIPTION
**Pull request by fabioseel**
- extends the video producer script to allow for synchronous generation of RAW, DECODED, AUGMENTED and VALUE_MASK videos.
- the value mask uses a more general analysis available in the analysis module, which will be helpful for #74 
- also removes a script that for whatever reason made it to be commited to the top level directory (shame on me)

### CODE REVIEW

File: runner/scripts/produce_video.py [hunk 1]
```diff
@@ -29,74 +33,144 @@
...
+    decoder_key = None
+    for response_key, response in responses.items():
+        if response[0].shape == obs["obs"].shape:
+            decoder_key = response_key
+            break
+
+    cur_frames: dict[VideoType, torch.Tensor] = {
+        VideoType.RAW: obs["obs"].detach(),
+        VideoType.AUGMENTED: normalized_obs["obs"].detach(),
+        VideoType.DECODED: responses[decoder_key][0].detach()
```
[the remark this answers is not in the document]
Comment (fabioseel) [reply runner/scripts/produce_video.py]: done
